src/library/scala/collection/Factory.scala

```diff
@@ -383,14 +383,29 @@ trait SpecificIterableFactory[-A, +C] extends Factory[A, C] {
   */
 trait MapFactory[+CC[_, _]] extends Serializable {
 
+  /**
+   * An empty Map
+   */
   def empty[K, V]: CC[K, V]
 
+  /**
+   * A collection of type Map generated from given iterable object.
+   */
   def from[K, V](it: IterableOnce[(K, V)]): CC[K, V]
 
+  /**
+   * A collection of type Map that contains given key/value bindings.
+   */
   def apply[K, V](elems: (K, V)*): CC[K, V] = from(elems)
 
+  /**
+   * The default builder for Map objects.
+   */
   def newBuilder[K, V]: Builder[(K, V), CC[K, V]]
 
+  /**
+   * Converts this to a Factory.
```

Comment:
Nothing's being converted.

Reply:
@dwijnand Thanks for your comment! So, should I remove this comment or do you have a suggestion?

Reply:
Oh, I guess it is converting `this`, my bad. 

I would check what docs are associated to this in other classes (for consistency) - I didn't look myself.

But I would look to say something like "The default Factory instance for maps.".

Reply:
Thanks! I didn't have strong confidence about the comment at all. So, I just applied your suggestion (that looks good to me too). If you or other maintainers have any other suggestions, let me know by using GitHub's change suggestion feature. I'm happy to apply the changes.

<img width="500" src="https://user-images.githubusercontent.com/19658/96457645-00392100-125b-11eb-9406-45cbc07036aa.png">